lib/ansible/modules/network/fortios/fortios_webfilter_profile.py

```diff
@@ -0,0 +1,999 @@
+#!/usr/bin/python
+from __future__ import (absolute_import, division, print_function)
+# Copyright 2018 Fortinet, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+# the lib use python logging can get it if the following is set in your
+# Ansible config.
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: fortios_webfilter_profile
+short_description: Configure Web filter profiles.
```

Comment:
Can you please raise a follow-up PR to add the company and product name into all the `short_descriptions`?

See https://docs.ansible.com/ansible/latest/modules/list_of_network_modules.html this is built from the `short_description`.

A single PR to update all `short_description` fof modules in `devel`  would fix this.

This allows people to find the modules when searching

Reply:
Sure. Excellent idea. I'll ensure new modules contain this info and I'll raise a PR for the ones already submitted. 
Thanks!!!